test/protractor.conf.js

```diff
@@ -9,15 +9,25 @@ require('ts-node').register({
 
 const E2E_BASE_URL = process.env['E2E_BASE_URL'] || 'http://localhost:4200';
 const config = {
-  // TODO(jelbourn): add back plugin for a11y assersions once it supports specifying AXS options.
   useAllAngular2AppRoots: true,
   specs: [ path.join(__dirname, '../e2e/**/*.e2e.ts') ],
   baseUrl: E2E_BASE_URL,
   allScriptsTimeout: 120000,
   getPageTimeout: 120000,
   jasmineNodeOpts: {
     defaultTimeoutInterval: 120000,
-  }
+  },
+
+  plugins: [
+    {
+      path: '../tools/axe-protractor/axe-protractor.js',
```

Comment:
I added a short comment for it.

> Runs the axe-core accessibility checks each time the e2e page changes and Angular is ready.

It basically means that the A11y check runs each time the e2e app changes its page.

Unfortunately we are not that flexible here, because the Protractor API does not provide much hooks (See https://github.com/angular/protractor/blob/master/docs/plugins.md)